<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventory</class>
 <widget class="QWidget" name="Inventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="radioButtonRedStone">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Red Stone</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonIce">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ice</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSand">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sand</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>INVENTORY</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numSand">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numSnow">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numDirt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGrass">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../inventories.qrc">:/inventory/Screen Shot 2022-04-18 at 11.15.30 AM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSand">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../inventories.qrc">:/inventory/Screen Shot 2022-04-18 at 11.18.28 AM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonGrass">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Grass</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numRedStone">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numIce">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelRedStone">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../inventories.qrc">:/inventory/Screen Shot 2022-04-18 at 11.24.22 AM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numStone">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDirt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../inventories.qrc">:/inventory/Screen Shot 2022-04-18 at 11.15.53 AM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelPumpkin">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../inventories.qrc">:/inventory/Screen Shot 2022-04-18 at 11.23.21 AM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSnow">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Snow</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStone">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../inventories.qrc">:/inventory/Screen Shot 2022-04-18 at 11.17.35 AM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelIce">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../inventories.qrc">:/inventory/Screen Shot 2022-04-18 at 11.22.52 AM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelSnow">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../inventories.qrc">:/inventory/snow.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numPumpkin">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDirt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Dirt</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonPumpkin">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Pumpkin</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonStone">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Stone</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numGrass">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../inventories.qrc"/>
 </resources>
 <connections/>
</ui>
